/**:
  ros__parameters:
    ###
    ### pcap file used to replay
    ###
    pcap_file: ""

    ###
    ### scale factor of timer wait, which emulates the interarrival times of lidar msgs
    ###
    pcap_wait_factor : 0.6

    ###
    ### lidar model selection:
    ### OS1-64, VLP-16, PDR-64
    ###
    model: "OS1-64"
    ns: "center"
    frame_id: "base_link"

    ###
    ### ip address and port number of each lidar
    ### ip address selection: "192.168.1.2", "192.168.1.3"
    ### lidar port selection: 7502, 7504
    ###
    ip_addr: "192.168.1.3"
    lidar_port: 7504

    ###
    ### using imu
    ###
    imu_port: 7503
    use_imu: false

    ###
    ### publish rate of point cloud
    ###
    period_ms: 100
    # rate: 10 # Hz

    used_channels: ### range: 0~63
      #START, END
      [0,     1,
       2,     3,
       4,     5,
       6,     7,
       8,     9,
       10,    11,
       12,    13,
       14,    15,
       16,    17,
       18,    19,
       20,    21,
       22,    23,
       24,    25,
       26,    27,
       28,    29,
       30,    31,
       32,    33,
       34,    35,
       36,    37,
       38,    39,
       40,    41,
       42,    43,
       44,    45,
       46,    47,
       48,    49,
       50,    51,
       52,    53,
       54,    55,
       56,    57,
       58,    59,
       60,    61,
       62,    63]

    used_azimuths: ### range: 0~1023
      #START, END
      [5,     1018]

    used_range: ### mm단위
      #START, END
      [300,   12000]